ab_tool/constants.py

```diff
@@ -5,4 +5,8 @@
 ADMINS = [ADMINISTRATOR, CONTENT_DEVELOPER, TEACHING_ASSISTANT, INSTRUCTOR]
 STAGE_URL_TAG = 'intervention_point_url_'
 DEPLOY_OPTION_TAG = 'is_canvas_page_for_track_'
-AS_TAB_TAG = 'open_as_tab_for_track_'
+
+NAME_CHAR_LIMIT = 256
```

Comment:
Just making sure that you ran these by Marlee and that they're not expected to change.  Otherwise, I could see a case for them being moved into settings.